plugins/plugin-site/src/pages/404.jsx

```diff
@@ -5,6 +5,9 @@ import {Helmet} from 'react-helmet';
 
 const title = 'Plugin page not found';
 const notFoundPage = 'pages/404.jsx';
+const prefillText = location.href.substring(
+    location.href.lastIndexOf('/') + 1, location.href.length
+);
 
 const NotFound = () => (
```

Comment:
```suggestion
const NotFound = ({ location }) => (
```

Reply:
in theory

Reply:
Yeah, makes sense. I completely forgot about SSR. I'll give this a try